libs/shared/spotlight/feature/src/lib/spotlight/spotlight.tsx

```diff
@@ -0,0 +1,107 @@
+import { useContext } from 'react'
+import { useNavigate } from 'react-router-dom'
+import { useIntercom } from 'react-use-intercom'
+import { IconEnum } from '@qovery/shared/enums'
+import {
+  SETTINGS_API_URL,
+  SETTINGS_CONTAINER_REGISTRIES_URL,
+  SETTINGS_GIT_REPOSITORY_ACCESS_URL,
+  SETTINGS_HELM_REPOSITORIES_URL,
+  SETTINGS_MEMBERS_URL,
+  SETTINGS_URL,
+  SETTINGS_WEBHOOKS,
+  USER_URL,
+} from '@qovery/shared/routes'
+import { Command, type CommandDialogProps, Icon } from '@qovery/shared/ui'
+import { SpotlightContext } from './spotlight-provider'
+
+export interface SpotlightProps extends Pick<CommandDialogProps, 'open' | 'onOpenChange'> {
+  organizationId: string
+}
+
+export function Spotlight({ organizationId, open, onOpenChange }: SpotlightProps) {
+  const navigate = useNavigate()
+  const { showMessages: showIntercomMessenger } = useIntercom()
+  const { quickActions } = useContext(SpotlightContext)
+
+  const iconClassName = 'text-brand-500 text-base text-center w-6'
+  const navigateTo = (link: string) => () => navigate(link)
+  const openExternalLink = (externalLink: string) => () => {
+    window.open(externalLink, '_blank')
+  }
+
+  return (
+    <Command.Dialog label="Console Spotlight" open={open} onOpenChange={onOpenChange}>
+      <Command.Input autoFocus placeholder="What do you need?" />
+      <Command.List>
+        <Command.Empty>
+          <div className="text-center px-3 py-6">
+            <Icon iconName="wave-pulse" className="text-neutral-350" />
+            <p className="text-neutral-350 font-medium text-xs mt-1">No result for this search</p>
+          </div>
+        </Command.Empty>
+        {quickActions.length > 0 && (
+          <Command.Group heading="Quick actions">
+            {quickActions.map(({ label, iconName, link }) => (
+              <Command.Item key={link} onSelect={navigateTo(link)}>
+                <Icon className={iconClassName} iconName={iconName} />
+                {label}
+              </Command.Item>
+            ))}
+          </Command.Group>
+        )}
+
+        <Command.Group heading="Settings">
+          <Command.Item onSelect={navigateTo(SETTINGS_URL(organizationId) + SETTINGS_CONTAINER_REGISTRIES_URL)}>
```

Comment:
Can we have `map` instead to repeat content from an array of object?

Reply:
I've done it but I'm not convinced that's better this way
https://github.com/Qovery/console/pull/1265/commits/ff905f0cf50ae69e79d548af051ed3d62151d234